packages/video_player/video_player/lib/video_player.dart

```diff
@@ -634,6 +633,32 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     return Caption.none;
   }
 
+  /// Optional field to specify a file containing the closed
+  /// captioning.
+  ///
+  /// This future will be awaited and the file will be loaded when
+  /// [initialize()] is called.
+  Future<ClosedCaptionFile>? get closedCaptionFile {
+    return _closedCaptionFileFuture;
+  }
+
+  /// Sets a closed caption file.
+  ///
+  /// If [closedCaptionFile] is null, closed caption file will be removed.
```

Comment:
s/closed caption file/closed captions/

Reply:
I didn't understand what you mean

Reply:
Sorry, I shouldn't use that shorthand in reviews.

Please replace "closed caption file" with "closed captions" in this comment.

Reply:
Just in the second sentence or both?